score_normalization_v2_for_toxicity.md

```diff
@@ -0,0 +1,174 @@
+# Perspective API score normalization: fixes for Toxicity
+
+On **Tuesday, 2017-08-15**, we plan to update the scores for the TOXICITY model
+in the Perspective API. All other models are not affected.
+
+**You may need to take action** if you use specific thresholds for the Toxicity
+model in your application (for example, holding high-scoring comments for human
+moderation, auto-approving low-scoring comments, or color-coding comments based
+on score), and you want to ensure that the threshold is unchanged.
+
+You **don't** need to take action if you only use the scores to sort or rank
+comments, though you may want to notify your users that the score values are
+changing.
+
+
+## When is it happening?
+
+We plan to start returning the updated Toxicity scores on **Tuesday,
+2017-08-15**. We will send a message to
+[perspective-announce](http://groups.google.com/forum/#!forum/perspective-announce)
+once the rollout is complete.
+
+
+## What should I do?
+
+To avoid using out-of-date score thresholds for Toxicity in your application,
+you can:
+* Consult the [toxicity score mappings](#score-mappings) below to update your
+  thresholds.
+* Configure your application for a
+  [controlled migration](#i-want-to-control-when-i-migrate).
+
+
+## What's changing and why?
+
+Our normalization process applies a version of [probability
+calibration](http://scikit-learn.org/stable/modules/calibration.html) to make
+the scores approximate probabilities. For example, so that a toxicity score of
+**0.80** can be interpreted as a belief that **80%** of people would consider
+the comment to be toxic.
+
+And yeah, sorry we're doing this again. Our first attempt at doing this wasn't
```

Comment:
I suggest adding a tl;dr summary. For example: "The latest calibration is gives less extreme scores than the previous calibration - it increases the scores that used to fall in the low end of the score range, and lowers the scores at the highest end.